docs/website/versioned_docs/version-3.0.0/development/devfile.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Devfile Reference
+sidebar_position: 30
+---
+
+## Description
+
+### Overview
+
+We implement the [Devfile 2.2.0 reference](https://devfile.io/docs/devfile/2.2.0/user-guide/api-reference/) in `odo`.
```

Comment:
Is that really true? I am under an impression that odo even works with 2.0.0 and 2.1.0 besides working with 2.2.0. 🤔